log/log.go

```diff
@@ -315,6 +316,17 @@ func (log *impl) Infof(format string, a ...interface{}) {
 	log.Info(fmt.Sprintf(format, a...))
 }
 
+// InfoObject logs an INFO level JSON-serialized object message.
+func (log *impl) InfoObject(msg string, obj interface{}) {
```

Comment:
Since it only has one caller, is it really worth adding this new method instead of just having that one caller (essentially) call `log.Infof("FinalizationCaaCheck: %s", json.Marshal(obj))`?

Alternatively, are there additional places where we can already take advantage of this new function, so it has more than one caller?

Reply:
I stashed this in the logger package for the sake of style consistency. This is just a non-audit version of the `AuditObject` method.

https://github.com/letsencrypt/boulder/blob/5c49231ea6bffe0b51e528944724f58f6df75eca/log/log.go#L340-L350